test/e2e/tests/swap-send/swap-send-test-utils.ts

```diff
@@ -266,7 +266,6 @@ export const getSwapSendFixtures = (
 ) => {
   const ETH_CONVERSION_RATE_USD = 3010;
   return {
-    driverOptions: { responsive: true },
```

Comment:
I would like to confirm with the original author before we delete this. @BZahory, was there a reason that this was originally written so that the dev tools would be open during the test run (here is where your PR first added this: https://github.com/MetaMask/metamask-extension/commit/d0a0037fcceb8ef7a18389d805daa0fe4f0e6b34#diff-f2647a24cd0065e584933d0fdc32081d6f28f75b7d1123533b91e02109d885fcR244_

Reply:
@danjm Sure, I agree with you we should confirm. Just to add BZahory may not reply as he is no longer working with the Extension. I am tagging the team @MetaMask/metamask-assets here for faster confirmation and this is my observation according to the PR in the screenshot and the test scenario where the extension is just clicked and hence I am thinking the size of the window is not mandatory.

Reply:
I can't currently see how the responsive screen size was needed, so I think it is fine to remove for now.

Reply:
Thanks Dan.